src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -1,13 +1,6 @@
 package carpet.script.value;
 
-import carpet.fakes.BrainInterface;
-import carpet.fakes.EntityInterface;
-import carpet.fakes.ItemEntityInterface;
-import carpet.fakes.LivingEntityInterface;
-import carpet.fakes.MemoryInterface;
-import carpet.fakes.MobEntityInterface;
-import carpet.fakes.ServerPlayerEntityInterface;
-import carpet.fakes.ServerPlayerInteractionManagerInterface;
+import carpet.fakes.*;
```

Comment:
Try to avoid collapsing all imports like that. I know the IDE does it and it's annoying, but please expand this